terraform/aws/analytical-platform-data-production/airflow/eks.tf

```diff
@@ -9,14 +16,18 @@ resource "aws_eks_cluster" "airflow_dev_eks_cluster" {
   ]
   version = "1.25"
 
+
   vpc_config {
-    subnet_ids          = aws_subnet.dev_private_subnet[*].id
-    public_access_cidrs = ["0.0.0.0/0"]
-    security_group_ids  = [var.dev_cluster_additional_sg_id]
+    subnet_ids             = aws_subnet.dev_private_subnet[*].id
```

Comment:
Would be good to chat about this change - looking at the subnets live I'm seeing `/24` ranges on them currently.

Reply:
amended an will push up change